Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.4
-MAINTAINER Michael de Wit <michael@drillster.com>
+FROM alpine:latest
```

Comment:
Using a `latest` version like this is very dangerous, you may get a version that doesn't work the way you want it to. I'd prefer to see a fixed version. Also, is changing the Alpine version required for this PR?

Reply:
this was changed for myself as i am planning to rewrite the plugin to my specific needs just didn’t realize the PR would use my last commit there. same goes for the maintainer thing. i’ll update the code to your request.